core/services/keeper/registry_synchronizer_core.go

```diff
@@ -93,13 +93,7 @@ func (rs *RegistrySynchronizer) Start(context.Context) error {
 				keeper_registry_wrapper.KeeperRegistryConfigSet{}.Topic():        nil,
 				keeper_registry_wrapper.KeeperRegistryUpkeepCanceled{}.Topic():   nil,
 				keeper_registry_wrapper.KeeperRegistryUpkeepRegistered{}.Topic(): nil,
-				keeper_registry_wrapper.KeeperRegistryUpkeepPerformed{}.Topic(): {
-					{},
-					{},
-					{
-						log.Topic(rs.job.KeeperSpec.FromAddress.Hash()),
```

Comment:
Did we confirm why this was added in the first place?

Reply:
comment below with image attached about race condition. i updated the query to handle that case